cime_config/tests.py

```diff
@@ -461,21 +461,21 @@
     "e3sm_scream_v1_lowres" : {
         "time"  : "01:00:00",
         "tests" : (
-            "ERP_D_Ln9.ne4_ne4.F2010-SCREAMv1",
+            "ERP_D_Ln9.ne4_oQU240.F2010-SCREAMv1.atmlndactive-rtm_off",
             "ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1",
```

Comment:
sanity check question: I think it would be good to keep at least one test using CICE (until CICE is completely removed from E3SM). That probably isn't accomplished very well from the aquaplanet ERS test above since that doesn't have any ice, right?

Reply:
That sounds good. We have ERP tests with Debug and non-debug mode, so let's just use CICE for one of them.